docs/notebooks/multi_objective_ehvi.pct.py

```diff
@@ -134,7 +134,7 @@ def build_stacked_independent_objectives_model(
 # We can now run the optimization loop
 
 # %%
-num_steps = 30
+num_steps = 1
```

Comment:
why only one step? for quick docs we do that automatically, for users we can leave more iterations no?

Reply:
back to normal values